common/src/main/java/com/scottlogic/deg/generator/fieldspecs/whitelist/NullDistributedSet.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.deg.generator.fieldspecs.whitelist;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class NullDistributedSet<T> implements DistributedSet<T> {
```

Comment:
what's the purpose of this class

Reply:
Acts as a distributed set including only `null`. This is because `FrequencyDistributedSet` rejects `null`, which inherits from the old set logic of the set rejecting the value `null`.

This is used in `StandardFieldValueSourceEvaluator` to append null values into the set if the field is marked at nullable.